11/Task11/Class2.cs

```diff
@@ -0,0 +1,9 @@
+﻿using System;
+
+namespace Task11
+{
+    public enum TransportType
+    {
+        plane, train, car, seaship, rivership

```

Comment:
В перечислениях принято элементы перечисления называть с большой буквы. Также принято каждый элемент указывать в отдельной строке. Так легче читается код.
```suggestion
        Plane, 
        Train, 
        Car, 
        Seaship, 
        Rivership
```